src/main/java/com/denizenscript/ddiscordbot/objects/DiscordCommandTag.java

```diff
@@ -0,0 +1,305 @@
+        this.bot = bot;
+        this.guild = guild;
+        this.guild_id = guild != null ? this.guild.getIdLong() : 0;
+        this.command = command;
+        this.command_id = this.command.getIdLong();
+    }
+
+    public DiscordConnection getBot() {
+        return DenizenDiscordBot.instance.connections.get(bot);
+    }
+
+    public Guild getGuild() {
+        if (guild != null) {
+            return guild;
+        }
+        if (bot == null) {
+            return null;
+        }
+        if (guild_id == 0) {
+            return null;
+        }
+        guild = getBot().client.getGuildById(guild_id);
+        return guild;
+    }
+
+    public Command getCommand() {
+        if (command != null) {
+            return command;
+        }
+        if (bot == null) {
+            return null;
+        }
+        if (getGuild() != null) {
+            command = getGuild().retrieveCommandById(command_id).complete();
+        }
+        else {
+            command = getBot().client.retrieveCommandById(command_id).complete();
+        }
+        return command;
+    }
+
+    public String bot;
+
+    public Guild guild;
+
+    public Command command;
+
+    public long guild_id;
+
+    public long command_id;
+
+    @Override
+    public AbstractFlagTracker getFlagTracker() {
+        return new RedirectionFlagTracker(getBot().flags, "__commands." + command_id);
+    }
+
+    @Override
+    public void reapplyTracker(AbstractFlagTracker tracker) {
+        // Nothing to do.
+    }
+
+    public static void registerTags() {
+        
+        AbstractFlagTracker.registerFlagHandlers(tagProcessor);
+
+        // <--[tag]
+        // @attribute <DiscordCommandTag.id>
+        // @returns ElementTag(Number)
+        // @plugin dDiscordBot
+        // @description
+        // Returns the ID of the command.
+        // -->
+        registerTag("id", (attribute, object) -> {
+            return new ElementTag(object.command_id);
+        });
+
+        // <--[tag]
+        // @attribute <DiscordCommandTag.name>
+        // @returns ElementTag
+        // @plugin dDiscordBot
+        // @description
+        // Returns the name of the command.
+        // -->
+        registerTag("name", (attribute, object) -> {
+            return new ElementTag(object.getCommand().getName());
+        });
+
+        // <--[tag]
+        // @attribute <DiscordCommandTag.description>
+        // @returns ElementTag
+        // @plugin dDiscordBot
+        // @description
+        // Returns the description of the command.
+        // -->
+        registerTag("description", (attribute, object) -> {
+            return new ElementTag(object.getCommand().getDescription());
+        });
+
+        // <--[tag]
+        // @attribute <DiscordCommandTag.options>
+        // @returns ListTag(MapTag)
+        // @plugin dDiscordBot
+        // @description
+        // Returns the option MapTags of the command.
```

Comment:
Should either explain its format or link to the doc that does

Reply:
It's the exact same list of maps that you provide it. Should I note that?

Reply:
Ye note that, and link to the relevant command meta doc